specification/common-types/resource-management/v3/privatelinks.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
     "title": "Common types",
-    "version": "3.0"
+    "version": "v3"
```

Comment:
I am not sure we should change the version numbers, what if people rely on them somehow?

Reply:
changed this to keep the version forma x.0